align_system/algorithms/adms.py

```diff
@@ -14,4 +15,5 @@
     'OracleADM': OracleADM,
     'OutlinesTransformersADM': OutlinesTransformersADM,
     'PersonaADM': PersonaADM,
+    'HybridRegressionADM': HybridRegressionADM
```

Comment:
No change needed here, but just FYSA "registering" an ADM in this file is no longer necessary with the Hydra configuration framework we're using now.  I have an MR to remove this file and older configs, but haven't merged it yet